[ UI - Dashboard ] Edit dashboard description fails

<!--

Thank you for reporting a bug in InfluxDB. 

* Please ask usage questions on the Influx Community site.
    * https://community.influxdata.com/
* Please add a :+1: or comment on a similar existing bug report instead of opening a new one.
    * https://github.com/influxdata/influxdb/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+is%3Aclosed+sort%3Aupdated-desc+label%3Akind%2Fbug+
* Please check whether the bug can be reproduced with the latest release.
* The fastest way to fix a bug is to open a Pull Request.
    * https://github.com/influxdata/influxdb/pulls

-->

__Steps to reproduce:__
List the minimal actions needed to reproduce the behavior.

1. Create a dashboard
2. In the dashboard card of the dashboards view click edit description
3. Add a description
4. Press ENTER

__Expected behavior:__
Describe what you expected to happen.

Expected the description to be added.

__Actual behavior:__
Describe What actually happened.

Error message shows "Could not update dashboard" 

Console log contains 

```
PATCH http://localhost:9999/api/v2/dashboards/undefined 400 (Bad Request)
```

__Screencasts__

![DashboardAddDescriptionFail01](https://user-images.githubusercontent.com/25881301/72908274-8d376400-3d35-11ea-8d85-1030a53482b3.gif)
![DashboardAddDescriptionFail02](https://user-images.githubusercontent.com/25881301/72908277-8d376400-3d35-11ea-8854-d9e362d8175a.gif)



__Environment info:__

* System info: Run `uname -srm` and copy the output here
```
Linux 4.15.0-74-generic x86_64
```
* InfluxDB version: Run `influxd version` and copy the output here
Detected in nightly builds 
```
ts=2020-01-22T09:58:21.846359Z lvl=info msg="Welcome to InfluxDB" log_id=0KVAreaW000 version=nightly commit=47b43bf98 build_date=2020-01-22T09:58:21Z
```
* Other relevant environment details: Container runtime, disk info, etc
Browser: __Chrome 79__

__Config:__
Copy any non-default config values here or attach the full config as a gist or file.

Started with end to end testing enabled

```
--e2e-testing=true
```

<!-- The following sections are only required if relevant. -->

__Logs:__
Include snippet of errors in log.

```
PATCH http://localhost:9999/api/v2/dashboards/undefined 400 (Bad Request)
n.request @ generatedRoutes.ts:1836
k @ generatedRoutes.ts:4633
(anonymous) @ thunks.ts:320
(anonymous) @ index.js:8
(anonymous) @ redux.js:449
handleUpdateDescription @ DashboardCard.tsx:173
(anonymous) @ ResourceCardEditableDescription.tsx:89
(anonymous) @ tslib.es6.js:99
(anonymous) @ tslib.es6.js:81
(anonymous) @ tslib.es6.js:73
Z @ tslib.es6.js:69
onKeyDown @ ResourceCardEditableDescription.tsx:86
s @ react-dom.production.min.js:14
h @ react-dom.production.min.js:15
(anonymous) @ react-dom.production.min.js:15
x @ react-dom.production.min.js:17
T @ react-dom.production.min.js:18
M @ react-dom.production.min.js:18
E @ react-dom.production.min.js:21
Dn @ react-dom.production.min.js:84
As @ react-dom.production.min.js:250
je @ react-dom.production.min.js:39
Tn @ react-dom.production.min.js:86
(anonymous) @ react-dom.production.min.js:251
t.unstable_runWithPriority @ scheduler.production.min.js:18
Fs @ react-dom.production.min.js:251
Ln @ react-dom.production.min.js:85
D.Object.isExtensible.Object.isExtensible.e.___hb.e.___hb @ honeybadger.js:466
thunks.ts:336 Error: id must have a length of 16 bytes
    at thunks.ts:326
(anonymous) @ thunks.ts:336
async function (async)
(anonymous) @ thunks.ts:320
(anonymous) @ index.js:8
(anonymous) @ redux.js:449
handleUpdateDescription @ DashboardCard.tsx:173
(anonymous) @ ResourceCardEditableDescription.tsx:89
(anonymous) @ tslib.es6.js:99
(anonymous) @ tslib.es6.js:81
(anonymous) @ tslib.es6.js:73
Z @ tslib.es6.js:69
onKeyDown @ ResourceCardEditableDescription.tsx:86
s @ react-dom.production.min.js:14
h @ react-dom.production.min.js:15
(anonymous) @ react-dom.production.min.js:15
x @ react-dom.production.min.js:17
T @ react-dom.production.min.js:18
M @ react-dom.production.min.js:18
E @ react-dom.production.min.js:21
Dn @ react-dom.production.min.js:84
As @ react-dom.production.min.js:250
je @ react-dom.production.min.js:39
Tn @ react-dom.production.min.js:86
(anonymous) @ react-dom.production.min.js:251
t.unstable_runWithPriority @ scheduler.production.min.js:18
Fs @ react-dom.production.min.js:251
Ln @ react-dom.production.min.js:85
D.Object.isExtensible.Object.isExtensible.e.___hb.e.___hb @ honeybadger.js:466
```

<!--

__Performance:__
Generate profiles with the following commands for bugs related to performance, locking, out of memory (OOM), etc.

```sh
# Commands should be run when the bug is actively happening.
# Note: This command will run for at least 30 seconds.
curl -o profiles.tar.gz "http://localhost:8086/debug/pprof/all?cpu=true"
curl -o vars.txt "http://localhost:8086/debug/vars"
iostat -xd 1 30 > iostat.txt
# Attach the `profiles.tar.gz`, `vars.txt`, and `iostat.txt` output files.
```
-->

--- a/ui/src/dashboards/components/dashboard_index/DashboardCard.tsx
+++ b/ui/src/dashboards/components/dashboard_index/DashboardCard.tsx
@@ -168,9 +168,9 @@ class DashboardCard extends PureComponent<Props> {
   }
 
   private handleUpdateDescription = (description: string) => {
-    const {onUpdateDashboard, params} = this.props
+    const {id, onUpdateDashboard} = this.props
 
-    onUpdateDashboard(params.dashboardID, {description})
+    onUpdateDashboard(id, {description})
   }
 
   private handleAddLabel = (label: Label) => {

